tests/tinytest.R

```diff
@@ -45,28 +31,14 @@ if (file.exists(connectionParameterFile)) source(connectionParameterFile)
 ## recall that we DO need a working Bloomberg connection...
 if (getOption("blpUnitTests", FALSE)) {
 
-    ## load the package
-    stopifnot(require(Rblpapi, quietly=TRUE))
-
     ## without this, we get (or used to get) unit test failures
     Sys.setenv("R_TESTS"="")
 
+    ## set an option the test files check for
     Sys.setenv("RunRblpapiUnitTests" = "yes")
 
-    ## Define tests
-    testSuite <- defineTestSuite(name="Rblpapi Unit Tests",
-                                 dirs=system.file("unitTests", package = "Rblpapi"),
-                                 testFuncRegexp = "^[Tt]est.+")
-
-    ## Run tests
-    tests <- runTestSuite(testSuite)
-
-    ## Print results
-    printTextProtocol(tests)
-
-    ## Return success or failure to R CMD CHECK
-    if (getErrors(tests)$nFail > 0) stop("TEST FAILED!")
-    if (getErrors(tests)$nErr > 0) stop("TEST HAD ERRORS!")
-    if (getErrors(tests)$nTestFunc < 1) stop("NO TEST FUNCTIONS RUN!")
+    ## standard test suite call provided tinytest is present
+    if (requireNamespace("tinytest", quietly=TRUE)) {
+        tinytest::test_package("td")
```

Comment:
Should be `test_package("Rblpapi")`, right?

Reply:
Hot damn.  Yes.  Copied and pasted and did not edit.  

Coincidentally, also see [my question here](https://stat.ethz.ch/pipermail/r-package-devel/2021q2/006852.html) 

Now fixed.